Vagrantfile

```diff
@@ -9,6 +9,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     config.vm.network :forwarded_port, guest: 8282, host: 8282
     config.vm.network :forwarded_port, guest: 80, host: 8000
     config.vm.network :forwarded_port, guest: 4443, host: 4443
+    config.vm.network :forwarded_port, guest: 777, host: 777
```

Comment:
Hmm, interesting.  I hadn't expected that the server cert would also be different.  I figured we'd just use the normal server cert and just use the custom CA for validating client certs.  In that case we could do it based on the path, since you can tell Nginx to do client cert validation, but only on requests to ``/jsonrpc`` or below.

This also seems like it should work though.